src/reducers/index.js

```diff
@@ -10,12 +10,21 @@ const initialState = {
 // by our action creator. What does the reducer
 // need to do with the count in each case?
 export default (state = initialState, action) => {
-  switch (action.type) {
-    case INCREMENT:
-    // Fill in the body of this case
-    case DECREMENT:
-    // Fill in the body of this case
-    default:
-      return state;
-  }
-};
+    switch (action.type) {
+        case INCREMENT:
+            // Fill in the body of this case
+            return {
+                ...state,
+                count: state.count + 1
+            };
+
+        case DECREMENT:
+            // Fill in the body of this case
+            return {
+                ...state,
```

Comment:
Do we need the properties off of state (`...state`) here? I'd say not since it's only `count`, and we're actually replacing it already.